cli/get.go

```diff
@@ -53,10 +53,9 @@ var getCmd = &cobra.Command{
 	// Example: appname + " get <accountName/alias>",
 	Args: cobra.ExactArgs(1),
 	RunE: func(cmd *cobra.Command, args []string) error {
-		ctx := context.Background()
-		client, err := newClient()
-		if err != nil {
-			return err
+		if HasTokenExpired(config.Tokens) {
```

Comment:
It's not clear how long this token expiry is just yet. The current method of operation means that users switching between accounts won't actually get prompted for MFA since we can surprisingly endlessly re-use the same id token and access token and exchange them for web sso tokens.

This is a downgrade in security from the previous mode of operation where you would have to MFA every time you authed into an account.

however, if the access/id token expiration is around a day, this is probably fine IMO - and in a way it's actually better for us because there are some customers who have built automation around keyconjurer and repeatedly spam us with login attempts that fail due to hitting a rate limit.